armadas/borg_assimilator_cube_35.json

```diff
@@ -32,7 +34,6 @@
       "shield_health": 60058000
     }
   },
-  "strength": 62808404,
```

Comment:
I realise the field wasn't in the schema, but why is the "strength" field removed here, because it can be calculated from the head stats?

Reply:
The strength is the sum of the head stats. That's why it is redundant to keep in store.